code_review_agent/cli.py

```
@@ -0,0 +1,245 @@
+            else:
+                console.print("✅ No active reviews")
+                
+        except Exception as e:
+            console.print(f"❌ Error: {e}")
+    
+    asyncio.run(_status())
+
+
+@app.command()
+def init(
+    directory: Optional[Path] = typer.Argument(None, help="Directory to initialize"),
```

Ruff (0.12.2)

Do not perform function call `typer.Argument` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)